debug/debugger.go

```diff
@@ -36,7 +36,7 @@ import (
 type Debugger interface {
 	// LaunchEval starts a new eval debug session with the given LaunchEvalProperties.
 	// The returned session is in a stopped state, and must be resumed to start execution.
-	LaunchEval(ctx context.Context, props LaunchEvalProperties) (Session, error)
+	LaunchEval(ctx context.Context, props LaunchEvalProperties, opts ...LaunchOption) (Session, error)
```

Comment:
This would a backward incompatible change, correct? We should make a note in the Changelog.